### PR TITLE
DAT-16097 DevOps :: Configure liquibase-maxdb repo to use build-logic

### PR DESCRIPTION
**Pull request by jandroav**
https://datical.atlassian.net/browse/DAT-16097

### CODE REVIEW

File: .github/workflows/release-published.yml [hunk 1]
```diff
@@ -1,80 +1,52 @@
...
-          echo "Publishing version ${{ steps.collect-data.outputs.version }} from ${{ steps.collect-data.outputs.tag }}"
+  release:
+    uses: liquibase/build-logic/.github/workflows/extension-release-published.yml@v0.5.2
+    secrets: inherit
 
-  deploy_maven:
-    name: Deploy to Maven
-    needs: [ setup ]
-    runs-on: ubuntu-latest
+  deploy_xsd:
```
Comment (jnewton03) [inline .github/workflows/release-published.yml:13]: @jandroav I don't think we need to publish XSDs for this

Comment (jandroav) [reply .github/workflows/release-published.yml:13]: Copy paste 🫣

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  build:
+    uses: liquibase/build-logic/.github/workflows/os-extension-test.yml@v0.5.2
+    secrets: inherit
+
+#  integration-test:
```
[the remark this answers is not in the document]
Comment (jandroav) [reply .github/workflows/test.yml:16]: They were commented in the original workflow

[the remark this answers is not in the document]
Comment (jnewton03) [reply .github/workflows/test.yml:16]: @filipelautert @kevin-atx who is the maintainer of this extensions that could re-enable the integration tests?

**Pull request merged**